doc/port.1.txt

```diff
@@ -129,7 +129,7 @@ The port command recognizes several global flags and options.
     Quiet mode, suppress informational messages to a minimum, implies -N
 
 -N::
-    Non-interactive mode, interactive questions are not asked 
+    Non-question mode, interactive questions are not asked (avoid typing yes)
```

Comment:
Shouldn't that be "no-questions"?

The phrase "avoid typing yes" is misleading, as in many cases the operation is aborted instead of choosing the "yes" option.